gap/LazyCategory.gi

```diff
@@ -781,7 +783,7 @@ InstallGlobalFunction( CAP_INTERNAL_COMPACT_NAME_OF_CATEGORICAL_OPERATION,
                 fi;
             fi;
         else
-            if IsInt( posOf ) then
+            if posOf <> fail then
```

Comment:
Did you need to change this line?

Reply:
I prefer the positive statement as the corner-cases might theoretically change in future :)

Reply:
Okay, should be fixed :-)